GetRawYieldsDplusDs.C

```diff
@@ -535,6 +535,10 @@ int GetRawYieldsDplusDs(int cent, bool isMC, TString infilename, TString cfgfile
                 massFitter->SetFixRatio2GausSigma(
                 hSigmaToFix1->GetBinContent(iPt+1)/hSigmaToFix2->GetBinContent(iPt+1));
             }
+            if(fixSigmaRatio){
+                massFitter->SetFixRatio2GausSigma(
+                hSigmaToFix1->GetBinContent(iPt+1)/hSigmaToFix2->GetBinContent(iPt+1));
+            }
```

Comment:
Actually, this is a duplicate of Lines 534-537

Reply:
indeed, I did not notice

Reply:
Fixed by #247